android/engine/src/test/java/org/smartregister/fhircore/engine/task/FhirCarePlanGeneratorTest.kt

```diff
@@ -783,6 +794,8 @@ class FhirCarePlanGeneratorTest : RobolectricTest() {
     val questionnaireResponses = planDefinitionResources.questionnaireResponses
     val resourcesSlot = planDefinitionResources.resourcesSlot
 
+    installToIgManager(planDefinition)
```

Comment:
We need to review all tests to make sure the objects are now split between the Knowledge manager and the Engine. Either that or rewrite each of the tests to be their own IG's (as they are supposed to)